stdlib/2and3/logging/handlers.pyi

```diff
@@ -195,12 +197,17 @@ class HTTPHandler(Handler):
 
 if sys.version_info >= (3,):
     class QueueHandler(Handler):
-        def __init__(self, queue: Queue) -> None: ...
+        if sys.version_info >= (3, 7):
+            def __init__(self, queue: Union[SimpleQueue, Queue]) -> None: ...
```

Comment:
This is now missing `__init__()` for Python <(3, 7). Maybe an easier solution is to define a type alias `_Queue`, which differs by Python version and use that instead having multiple `__init__()` versions.

Reply:
Crumbs, I clearly missed the `else` there. Good catch! I wonder why the tests didn't, though.

I'll stick with the same pattern as already used with `QueueListener` here for now.